src/DynamoCoreWpf/UI/GuidedTour/GuidedTourResources.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Wpf.UI.GuidedTour
+{
+    public class GuidedTourResources
+    {
+        internal static string LoadContentFromResources(string name, Assembly assembly)
```

Comment:
I think it would be better if you could move this shared code to a shared location, like DynamoUitlls or DynamoCore so both extensions could use it and we won't have to keep this code in sync in two places.

Reply:
Hi @mjkkirschner 
I got your point about moving the code to a shared location but in that way we could affect the DocumentationBrowserViewModel class and then we will need to check if any test or functionality related to DocumentationBrowser was affected.

Since this PR is blocking in some way the implementation of the next tasks related to the packages tour, is that ok if we create a new Jira taks for dealing with this problem? or you consider that we should fix this code share location problem?
Please let me know your comments.

FYI: @QilongTang @filipeotero 

Thanks

Reply:
Thanks @mjkkirschner , we did notice this and will file a tech debt for it. Since this PR is blocking @RobertGlobant20 to implement follow up steps, we may just have to make follow up PRs for potential improvements.

Reply:
@filipeotero Can you file this as well?